dotCMS/src/main/java/com/dotcms/publishing/PublisherAPIImpl.java

```diff
@@ -97,6 +102,11 @@ public PublishStatus publish ( PublisherConfig config, PublishStatus status ) th
                         bundler.generate( bundleRoot, bs );
                         Logger.info(this, "End of Bundler: " + clazz.getSimpleName());
                     }
+
+                    currentStatusHistory.setBundleEnd(new Date());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)